mkdocs.yml

```diff
@@ -31,12 +31,15 @@ nav:
       - Math: api/math.md
       # - PySpark: api/pyspark.md  # will be added back later
       - Timeseries: api/timeseries.md
-      - Utils: api/utils.md
+      # - Utils: api/utils.md
       - XArray: api/xarray.md
   - Changelog: CHANGELOG.md
   - Authors: AUTHORS.md
 
 plugins:
+  - exclude:
+      glob:
+        - api/utils.md
   - search
```

Comment:
Rather than introducing an additional dependency (mkdocs-exclude), how about we just remove the `api/utils.md` file entirely? Would that work?

Reply:
so we move relevant files out of utils.md? dont know if that would work

Reply:
we probably should move some of the relevant functions out of `utils`, like `unionize_dataframe_categories` into its own section, or a section of useful tools. open to suggestions on how that should look like

Reply:
ah now i remember why i used `exclude` - mkdocs builds everything, i then have to exclude the ones I dont want